### PR TITLE
Silkscreen on the pads

### PR DESCRIPTION
**Pull request by razykov**
Problem with silkscreen on the pads of electrolytic capacitors.

silkscreen_bug

Now it's fixed:

new_footprint

### CODE REVIEW

**Comment by SchrodingersGat** [comment]
Hi @razykov it's great that someone has finally taken the initiative to fix this! :)

This is a great job you have done thus far. 

Now that you "touched it last", there are some other key improvements that should be made.

## Setback

I would recommend increasing the setback of the silkscreen so that it does encroach on the default pad setback

setback

## Orientation

IPC standards require pin-1 to be on the LEFT. 

[IPC](http://pcbget.ru/Files/Standarts/IPC_7351.pdf) <- Refer to p66

Please rotate each part 180 degrees (and the 3D model too)

cap

## Fabrication Layer

IPC also requires a simple mechanical outline of the component on the F.Fab (fabrication) layer. This line should be 0.15mm width. You should also include a polarity reference on this layer. See below:

f fab

This is a lot to dump on you, but I am confident you will rise to the challenge ;)

**Comment by razykov** [comment]
Thank you, @SchrodingersGat , for your advice.
I tried to fix it.

**Comment by razykov** [comment]
Few screenshots for quick viewing:
screenshot_20161006_173106
screenshot_20161006_173138
screenshot_20161006_173211
screenshot_20161006_173328

**Comment by eldarkg** [comment]
Link to #9

**Comment by eldarkg** [comment]
It is look good

**Comment by razykov** [comment]
Yes. 0.0127mm.

**Comment by eldarkg** [comment]
Why you don't use the metric grid?

**Comment by razykov** [comment]
It's not my choice. I don't have change the size of silkscreen and pads. If you notice a violation of the standard then write it to me.

**Comment by eldarkg** [comment]
@razykov I see that in IPC-SM-782A used metric sizes. But this standard doesn't show this housing. Manufacturers like Vishay recomend sizes for soldering in the metric system.

**Comment by SchrodingersGat** [comment]
@eldarkg  are you referring to the size of the pads, or something else? Perhaps you could provide an example of what you mean.

**Comment by eldarkg** [comment]
@SchrodingersGat Yes, I am referring to sizes of pads and to sizes between pads.

**Comment by razykov** [comment]
Changed the size of pads in accordance with [that table](http://www.nichicon.co.jp/english/products/pdfs/e-ch_ref.pdf)

File: c_elec_10x10.5.kicad_mod [hunk 1]
```diff
@@ -1,12 +1,18 @@
-(module c_elec_10x10.5 (layer F.Cu) (tedit 57F65A59)
-  (descr "SMT capacitor, aluminium electrolytic, 10x10.5")
+(module c_elec_10x10.5 (layer F.Cu) (tedit 57FA4AE4)
+  (descr "SMT capacitor, aluminium electrolytic, 10x10")
```
Comment (eldarkg) [inline c_elec_10x10.5.kicad_mod]: Is this intentionally? From 10x10.5 to 10x10 and etc.

File: c_elec_10x7.7.kicad_mod [hunk 2]
```diff
@@ -1,28 +1,28 @@
-(module c_elec_10x7.7 (layer F.Cu) (tedit 57F6580C)
-  (descr "SMT capacitor, aluminium electrolytic, 10x7.7")
+(module c_elec_10x7.7 (layer F.Cu) (tedit 57FA4B25)
+  (descr "SMT capacitor, aluminium electrolytic, 10x10")
```
Comment (eldarkg) [inline c_elec_10x7.7.kicad_mod]: The same

**Comment by razykov** [comment]
No. It bug but now I'm fix it.

**Comment by razykov** [comment]
Thank you!!!

**Comment by eldarkg** [comment]
@razykov I'm happy to help ) Good work

**Comment by SchrodingersGat** [comment]
@razykov thanks this is a great contribution :)

**Pull request merged**